app/javascript/dashboard/components/widgets/conversation/ConversationBox.vue

```diff
@@ -75,7 +75,7 @@ export default {
 };
 </script>
 <style lang="scss" scoped>
-@import '~dashboard/assets/scss/app.scss';
+@import '~dashboard/assets/scss/woot';
```

Comment:
This would include the entire CSS under the scope of this component. Can we include mixin or space helper files only?